modules/apps/powwow/powwow-portlet/docroot/WEB-INF/src/com/liferay/powwow/meetings/portlet/MeetingsPortlet.java

```diff
@@ -936,6 +980,26 @@ protected CalendarBooking updateCalendarBooking(
 		return calendarBooking;
 	}
 
+	private void _addPowwowMeetingOccurrence(
+		JSONObject occurrenceJSONObject, long powwowMeetingId, long calendarBookingId) throws Exception {
+
+		CalendarBooking calendarBooking = CalendarBookingLocalServiceUtil.fetchCalendarBooking(calendarBookingId);
```

Comment:
because this method is called in a loop, we should be able to reuse the instance like `calendarBooking` and similar objects.